pkg/leader/election.go

```diff
@@ -35,11 +36,11 @@ func NewElection(clientset kubernetes.Interface, id, namespace string) *Election
 		RetryPeriod:     2 * time.Second,
 		Callbacks: leaderelection.LeaderCallbacks{
 			OnStartedLeading: func(ctx context.Context) {
-				log.Println("Started leading")
+				log.Printf("%v started leading", id)
```

Comment:
nit: `%v` does the job but it is more common to use `%s` for string or `%q` to have a quoted string

Reply:
I didn't know about %q that might actually be nice here.